dwave/system/testing.py

```diff
@@ -114,19 +114,42 @@ class MockDWaveSampler(dimod.Sampler, dimod.Structured):
     edgelist = None
     properties = None
     parameters = None
-
-    # by default, use ExactSolver for problems up to size (inclusive):
+    # In principle this class can be wrapped and additional parameters emulated
+    # (see shimming-tutorial)
```

Comment:
can probably remove comment since it's not relevant to this repo

Reply:
Almost all these errors you are pointing out were introduced by me, sorry @AndyZzzZzzZzz !